R/input-utils.R

```diff
@@ -7,7 +7,7 @@ controlLabel <- function(controlName, label) {
 # refers to values. Below is a function for backward compatibility.
 validateSelected <- function(selected, choices, inputId) {
   # drop names, otherwise toJSON() keeps them too
-  selected <- unname(selected)
+  selected <- as.character(unname(selected))
```

Comment:
`unname()` isn't actually needed here. Maybe just use `as.character()` and update the comment to mention that `as.character()` does two things for us here: coerces type, and drops name.